apps/SageAccountWeb/src/components/ConfigureEmail.tsx

```diff
@@ -99,6 +100,7 @@ export const ConfigureEmail = (props: ConfigureEmailProps) => {
     try {
       await SynapseClient.deleteEmail(accessToken, email)
       refetchCurrentProfile()
+      displayToast('Email has been successfully removed', 'success')
```

Comment:
Noticed that there was no notification when an email was removed

Reply:
Could consider adding a notification to `changePrimaryEmail` when the primary email was successfully changed, but up to you!

Reply:
Good idea